tfe/data_source_workspace_test.go

```diff
@@ -36,6 +36,10 @@ func TestAccTFEWorkspaceDataSource_basic(t *testing.T) {
 						"data.tfe_workspace.foobar", "terraform_version", "0.11.1"),
 					resource.TestCheckResourceAttr(
 						"data.tfe_workspace.foobar", "working_directory", "terraform/test"),
+					resource.TestCheckResourceAttr(
+						"data.tfe_workspace.foobar", "file_triggers_enabled", "true"),
+					resource.TestCheckResourceAttr(
+						"data.tfe_workspace.foobar", "trigger_prefixes", "[\"/modules\", \"/shared\"]"),
```

Comment:
Should this really be a string which describes an array? That looks wrong to me.

Reply:
Totally wrong. Fixed now.